src/scss/custom/_form-password-input.scss

```diff
@@ -0,0 +1,5 @@
+.form-password-input {
```

Comment:
Can you use a variable for this? We've got a bunch of spacer utils which use REM instead of fixed pixel values.